src/components/button/index.js

```diff
@@ -295,16 +300,15 @@ export default class Button extends BaseComponent {
   }
 
   getIconStyle() {
-    const {size, disabled, iconStyle: propsIconStyle} = this.props;
+    const {size, disabled, iconStyle: propsIconStyle, iconOnRight} = this.props;
     const iconStyle = {
       tintColor: this.getLabelColor(),
     };
 
-    if ([Button.sizes.large, Button.sizes.medium].includes(size)) {
-      iconStyle.marginRight = 8;
-    } else {
-      iconStyle.marginRight = 4;
-    }
+    const marginSide =
+      [Button.sizes.large, Button.sizes.medium].includes(size) ? 8 : 4;
+    iconOnRight ? // eslint-disable-line
```

Comment:
@RazLandau 
there's a reason for lint rules (: 
I prefer an if statement just for readability. 
change it and ill merge the change

Reply:
Changed. Thanks for the quick feedback!